feature/bgp/policybase/otg_tests/actions-MED_LocPref_prepend_flow-control/README.md

```diff
@@ -0,0 +1,599 @@
+# RT-1.32: BGP policy actions - MED, LocPref, prepend, flow-control
+
+## Summary
+
+- Verify abilty to set MED to fixed value in export and import policy
+- Verify abilty to increment MED by fixed value in export and import policy
+- Verify abilty to set Local Preference to fixed value in export and import policy
+- Verify abilty to prepend AS path with 10 additional repetitions of local ASN in export and import policy
+- Verify abilty to prepend AS path with 10 additional repetitions of configured ASN in export and import policy
+- verify ```NEXT-STATEMENT``` flow-control action
+- Applicable to both IPv4 and IPv6 BGP neighbors
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed
+
+## Procedure
+
+#### Initial Setup:
+
+*   Connect DUT port-1, 2 to ATE port-1, 2
+*   Configure IPv4/IPv6 addresses on the ports
+*   Create an IPv4 networks i.e. ```ipv4-network-1 = 192.168.10.0/24``` attached to ATE port-1
+*   Create an IPv6 networks i.e. ```ipv6-network-1 = 2024:db8:128:128::/64``` attached to ATE port-1
+*   Create an IPv4 networks i.e. ```ipv4-network-2 = 192.168.20.0/24``` attached to ATE port-2
+*   Create an IPv6 networks i.e. ```ipv6-network-2 = 2024:db8:64:64::/64``` attached to ATE port-2
+*   Configure IPv4 and IPv6 iBGP between DUT Port-1 and ATE Port-1
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/config
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/
+    *   Advertise ```ipv4-network-1 = 192.168.10.0/24``` and ```ipv6-network-1 = 2024:db8:128:128::/64``` from ATE to DUT over the IPv4 and IPv6 eBGP session on port-1 with:
+        * MED = 50
+        * Local Preference = 50
+*   Configure IPv4 and IPv6 eBGP between DUT Port-2 and ATE Port-2
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/config
+    *   /network-instances/network-instance/protocols/protocol/bgp/global/afi-safis/afi-safi/config/
+    *   Advertise ```ipv4-network-2 = 192.168.20.0/24``` and ```ipv6-network-2 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 eBGP session on port-2 with:
+        * MED = 50
+        * Local Preference = 50
```

Comment:
I suggest clarifying that the ATE is setting these BGP attributes (vs. the DUT which is what is done in the tests below).

```suggestion
    *   Advertise ```ipv4-network-2 = 192.168.20.0/24``` and ```ipv6-network-2 = 2024:db8:64:64::/64``` from ATE to DUT over the IPv4 and IPv6 eBGP session on port-2.  The ATE should advertise both prefixes with:
        * MED = 50
        * Local Preference = 50
```